deis-cleanup.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+
+require "open3"
+require "csv"
+
+KEEP_LAST_VERSIONS = ENV["KEEP_LAST_VERSIONS"].to_i || 2
+DRY_RUN = ENV["DRY_RUN"] || false
```

Comment:
`!obj.nil?` doesn't solve issue either :/. What do you think about `YAML.load(ENV.fetch("DRY_RUN", "false"))`?